src/main/scala/no/ndla/imageapi/service/ConverterService.scala

```diff
@@ -59,6 +61,63 @@ trait ConverterService {
         domainImageMetaInformation.captions.map(asApiCaption))
     }
 
+    def asApiImageMetaInformationWithApplicationUrlAndSingleLanguage(domainImageMetaInformation: domain.ImageMetaInformation, language: String): Option[api.ImageMetaInformationSingleLanguage] = {
+      val rawPath = ApplicationUrl.get.replace("/v2/images/", "/raw/")
+      asApiImageMetaInformationWithSingleLanguage(domainImageMetaInformation, language, Some(ApplicationUrl.get), Some(rawPath))
+    }
+
+    def asApiImageMetaInformationWithDomainUrlAndSingleLanguage(domainImageMetaInformation: domain.ImageMetaInformation, language: String): Option[api.ImageMetaInformationSingleLanguage] = {
+      asApiImageMetaInformationWithSingleLanguage(domainImageMetaInformation, language, Some(ImageApiProperties.ImageApiUrlBase.replace("v1", "v2")), Some(ImageApiProperties.RawImageUrlBase))
+    }
+
+    private def asApiImageMetaInformationWithSingleLanguage(domainImageMetaInformation: domain.ImageMetaInformation,
+                                                            language: String,
+                                                            apiBaseUrl: Option[String] = None,
```

Comment:
Om det er mulig så unngå å bruk Option som et input-parameter til en funksjon, og heller ikke gi default-verdier til parametre med mindre det er behov for det. Om du ser det er gjort andre steder må du gjerne fikse der også ;)